target/linux/ramips/mt7621/base-files/etc/board.d/01_leds

```diff
@@ -155,6 +155,9 @@ xiaomi,redmi-router-ac2100)
 youhua,wr1200js)
 	ucidef_set_led_netdev "internet" "INTERNET" "green:wan" "wan"
 	;;
+sztozed,zlt-s12-pro)
+	ucidef_set_led_netdev "wifi" "WiFi" "blue:wifi" "wlan0"
+	;;
```

Comment:
This section can be removed, and migrated in the device tree under the gpio-leds node for wifi led